hooks/tk-multi-workfiles2/scene_operation_tk-nuke.py

```diff
@@ -331,10 +331,20 @@ def _scene_operation_hiero_nukestudio(
         """
         import hiero
 
+        engine = sgtk.platform.current_engine()
+        engine_context = engine.context
+
         if operation == "current_path":
             # return the current script path
             project = self._get_current_hiero_project()
             curr_path = project.path().replace("/", os.path.sep)
+            if curr_path:
+                # switch file-save window context to selected project context
+                project = self._get_current_hiero_project()
+                tag_object = self.get_tag_object(project)
+                context_str = tag_object.note()
+                selected_project_context = engine_context.deserialize(context_str)
```

Comment:
This is an expensive operation, please wait for https://github.com/mbrainerd/tk-core/pull/39 and update the code

Reply:
Addressed comment.